agent/api/task/task.go

```diff
@@ -562,6 +586,83 @@ func (task *Task) initializeEFSVolumes(cfg *config.Config, dockerClient dockerap
 	return nil
 }
 
+func buildVolumeNameForExecAgentBinary(binaryName string) string {
+	return fmt.Sprintf("%s-%s", internalExecAgentNamePrefix, binaryName)
+}
+
+// initializeExecAgentResources specifies the necessary volumes and mount points in all of the task containers in order for the
+// exec agent to run upon container start.
+func (task *Task) initializeExecAgentResources(cfg *config.Config) error {
```

Comment:
this should be in task_linux.go file (since its linux only feature) same for the unit tests

Reply:
Good catch. Fixed.